packages/playground/windows/playground-composition/Playground-Composition.cpp

```diff
@@ -505,10 +500,43 @@ LRESULT CALLBACK WndProc(HWND hwnd, UINT message, WPARAM wparam, LPARAM lparam)
 
 constexpr PCWSTR c_windowClassName = L"MS_REACTNATIVE_PLAYGROUND_COMPOSITION";
 
+// The compositor could be running on a background or foreground thread.
+// Office runs it on a background thread, so its useful to be able to test that case
+void CreateCompositor(bool background, WindowData *windowData) {
+  DispatcherQueueOptions options{
+      sizeof(DispatcherQueueOptions), /* dwSize */
+      background ? DQTYPE_THREAD_CURRENT : DQTYPE_THREAD_DEDICATED, /* threadType */
```

Comment:
Is this backwards? Old behavior used `DQTYPE_THREAD_CURRENT`, but that should map to !background.

Reply:
Oops - Thanks